test.py

```diff
@@ -0,0 +1,55 @@
+import unittest
+import winnowing
+from os import path
+
+
+# getting the path of file 
+def setUp(file):
```

Comment:
This setup function should be inside the class. Also store the file paths as class attribute